src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -102,6 +112,9 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
                     button.render(container);
                 }
             });
+            if (messageContainer && messagingContainer && ppsdkFeatureOn) {
```

Comment:
and condition also will be according to previous comment (we already checked messagingContainer) so
```
if (isMessageContainerAvailable && ppsdkFeatureOn) {
```

Reply:
I set that kind of condition because i need to pass string as container but tslint don't know it is exists and think type of container is undefined.

Reply:
I know that we have already checked messagingContainer here `const isMessageContainerAvailable = Boolean(messagingContainer && document.getElementById(messagingContainer));` But tslint don't pay attention on it